spec/std/path_spec.cr

```diff
@@ -10,21 +10,26 @@ private def it_normalizes_path(path, posix = path, windows = path, file = __FILE
   assert_paths(path, posix, windows, "normalizes", file, line, &.normalize)
 end
 
-private def it_expands_path(path, posix, windows = posix, *, base = nil, env_home = nil, expand_base = false, file = __FILE__, line = __LINE__)
+private def it_expands_path(path, posix, windows = posix, *, base = nil, env_home = nil, expand_base = false, home = false, file = __FILE__, line = __LINE__)
   assert_paths(path, posix, windows, %((base: "#{base}")), file, line) do |path|
     prev_home = ENV["HOME"]
 
     begin
       ENV["HOME"] = env_home || (path.windows? ? HOME_WINDOWS : HOME_POSIX)
 
       base_arg = base || (path.windows? ? BASE_WINDOWS : BASE_POSIX)
-      path.expand(base_arg.not_nil!, expand_base: !!expand_base)
+      #      home = Path.home if home == true
+      path.expand(base_arg.not_nil!, expand_base: !!expand_base, home: home)
     ensure
       ENV["HOME"] = prev_home
     end
   end
 end
 
+private def it_expands_path_with_home(path, posix, windows = posix, *, base = nil, env_home = nil, expand_base = false, file = __FILE__, line = __LINE__)
```

Comment:
Does this need to be a separate helper method? The difference at the callsite is `_with_home` vs. adding a `, home: true` argument.

Reply:
Can the `Bool` argument for home stay?  If not I need to understand why the commented line in the review I posted below causes spec failures.

Reply:
Also, the original documentation for home expansion was incorrect.